### ISSUE TITLE
KFVPDKSD Log import doing nothing

### ISSUE THREAD

**Issue by JohnBaumb** [KalebKE/ME7Tuner#16]
I'm not positive if there is something wrong with my log file, but nothing is happening on import.

I assume it should populate the "Maximum boost (input)" line and the graph.

Somehow a table is being generated, but I'm not sure if I trust it w/o confirmation that files were loaded successfully.

It also says it needs a directory of logging, is a single long log okay?

log attached below, same bin (for the most part) /xdf as https://github.com/KalebKE/ME7Tuner/issues/15 (ps you need to add a "(" when searching for KFVPDSKSD)

[me7tuner_KFPDKSD_Logging_20220327_225046.csv](https://github.com/KalebKE/ME7Tuner/files/8360141/me7tuner_KFPDKSD_Logging_20220327_225046.csv)

**Comment by KalebKE** [comment]
This looks reasonable to me. It should populate the table and the plot. I assume you are not seeing what I am seeing?

A single log is acceptable for KFVPDKSD, but many logs with the same tune with varying ambient temperatures is better.

Your boost dips at 5000rpm and it results in some odd values, but you can patch those up by hand and log again.

![Screen Shot 2022-03-28 at 6 33 27 PM](https://user-images.githubusercontent.com/3472406/160509335-fd44249c-12a1-4d4e-8676-ac576fb25973.png)

**Comment by JohnBaumb** [comment]
Amazing, it must have been some problem with the instance. I restarted program and tested again, functioned fine this time. I've already corrected that boost dip at 5k. 

It would be nice to print out beneath the directory if logs (and maybe how many lines) were loaded total and parsed successfully. 

I'll keep testing and report any issues I find, looks like a great addition to the toolbox. I'll go ahead and close this, appreciate you looking into it, keep up the good work!
